packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -491,6 +500,52 @@ Future<XcodeBuildResult> buildXcodeProject({
   }
 }
 
+void updateShorebirdYaml(BuildInfo buildInfo, String xcarchivePath) {
+  final File shorebirdYaml = globals.fs.file(
+    globals.fs.path.join(
+      xcarchivePath,
+      'Products',
+      'Applications',
+      'Runner.app',
```

Comment:
Haven't heard any reports of this yet, but I really wonder when we'll get our first error report caused by someone renaming "Runner"